test/cmd/deployments.sh

```diff
@@ -29,6 +37,20 @@ oc describe deploymentConfigs test-deployment-config
 [ "$(echo "OTHER=foo" | oc env dc/test-deployment-config -e - --list | grep OTHER=foo)" ]
 [ ! "$(echo "#OTHER=foo" | oc env dc/test-deployment-config -e - --list | grep OTHER=foo)" ]
 [ "$(oc env dc/test-deployment-config TEST=bar OTHER=baz BAR-)" ]
+[ ! "$(oc env -f test/integration/fixtures/test-deployment-config TEST=VERSION -o yaml | grep v1beta3)" ]
+oc env dc/test-deployment-config TEST=foo OTHER=bar ANOTHER=env
+# Update an env
+oc env dc/test-deployment-config TEST=update --list | grep TEST=update
```

Comment:
I think a test like this would suffice:
1. add env vars A=a B=b C=c D=d E=e F=f G=g
2. verify values `A=a B=b C=c D=d E=e F=f G=g` (may need to do individual checks)
3. issue a single env command like `H=h G- E=updated C- A-`
4. verify values `B=b D=d E=updated F=f H=h` (may need to do individual checks)
5. verify absence of values `A C G`